src/interfaces/custom_config.ts

```diff
@@ -0,0 +1,18 @@
+export interface CustomConfig {
```

Comment:
Would "CustomDefinitions" be a better name for this interface? Then we will have "CustomDefinitionsLoader" instead of "ConfigLoader".